fpo-api/api/views/SubmitTicketResponseView.py

```diff
@@ -0,0 +1,178 @@
+import json
+import logging
+
+from django.conf import settings
+from django.http import HttpResponseBadRequest, HttpResponseForbidden
+from django.template.loader import get_template
+from django.utils import timezone
+
+from rest_framework.views import APIView
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from api.auth import (
+    grecaptcha_verify,
+    grecaptcha_site_key,
+)
+
+from api.models import TicketResponse, PreparedPdf
+from api.pdf import render as render_pdf
+from api.send_email import send_email
+from api.utils import generate_pdf
+
+from datetime import date, datetime  # For working with dates
+
+LOGGER = logging.getLogger(__name__)
+
+class SubmitTicketResponseView(APIView):
+    def get(self, request: Request, name=None):
+        key = grecaptcha_site_key()
+        return Response({"key": key})
+
+    def post(self, request: Request, name=None):
+        check_captcha = grecaptcha_verify(request)
+        if not check_captcha["status"]:
+            return HttpResponseForbidden(text=check_captcha["message"])
+
+        #############################################################
+        #  Adding different pdf form logic: Jul 3, 2020
+        data = json.loads(request.body)
```

Comment:
The request JSON is being loaded twice in this method (also as request.data which is preferred). It should only be loaded once for consistency.

The code to add the date to the payload, select a template, and prepare the PDF should be moved into a separate method and called inside the try block at the end of this method.